lib/element/locator.js

```diff
@@ -62,9 +62,12 @@ class LocateElement {
    * @param {Boolean} [returnSingleElement]
    * @return {Promise}
    */
-  async findElement({element, commandName, id, transportAction = 'locateMultipleElements', returnSingleElement = true}) {
```

Comment:
Some changes are purely "aesthetic" because I removed these defaults here to be more explicit in its usages.
I can revert those and only apply the required changes where needed.
(I mainly did this to properly understand what's going on 😬 )

Reply:
Better to remove the style changes. The code style should not be changed and it's also harder to review the overall PR.